docker/entrypoint-integration-tests.sh

```diff
@@ -215,20 +211,6 @@ else
         fail $test
     fi
 
-    test="Read only user profile test"
-    echo "Preparing configuration: USER_PROFILE_EDITABLE=True"
-    echo "USER_PROFILE_EDITABLE=Frue" > /app/dojo/settings/local_settings.py
-    reload
-
-    echo "Running $test"
-    if python3 tests/user_standard_test.py ; then
-        success $test
-    else
-        fail $test
-    fi
-    echo "USER_PROFILE_EDITABLE=True" > /app/dojo/settings/local_settings.py
-    reload
-
```

Comment:
these should not be removed, but the test should be added to the GHA

Reply:
I can add this back and add the file to the GitHub Action yaml, but the file does not exist so it will be unable to complete the action.

I opened this issue for that missing file: #5400

Reply:
ok yeah those #5400 and #5401 must be resolved first, but better to remove the changes here so we can merge the rest of your PR

Reply:
This tests are already in GHA, but I missed to execute local tests...
https://github.com/DefectDojo/django-DefectDojo/blob/master/.github/workflows/integration-tests.yml#L100
https://github.com/DefectDojo/django-DefectDojo/blob/master/tests/user_test.py#L182